content/pt-br/community/code-of-conduct.md

```diff
@@ -1,27 +1,30 @@
 ---
-title: Community
+title: Código de Conduta da Comunidade Kubernetes
 layout: basic
 cid: community
-css: /css/community.css
+community_styles_migrated: true
 ---
 
-<div class="community_main">
-<h1>Código de Conduta da Comunidade Kubernetes</h1>
-
+<div class="community-section" id="cncf-code-of-conduct-intro">
+<p>
 Kubernetes segue
-<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Código de Conduta</a>.
+<a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md">CNCF Código de Conduta</a>.
 O texto do CNC CoC é reproduzido abaixo, a partir de
-<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+<a href="https://github.com/cncf/foundation/blob/fff715fb000ba4d7422684eca1d50d80676be254/code-of-conduct.md">commit fff715fb0</a>.
 Se você perceber que isso está desatualizado, por favor
 <a href="https://github.com/kubernetes/website/issues/new">abra um bug</a>.
+</p>
+
 
-Se você notar uma violação do Código de Conduta em um evento ou meetup,
-ou em outro meio de comunicação, entre em contato com
-<a href="https://git.k8s.io/community/committee-code-of-conduct">Comitê de Código de Conduta da Comunidade Kubernetes</a>. 
-Você pode nos contatar por e-mail em<a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
+<p>
+Se você notar uma violação do Código de Conduta em um evento ou reunião, no
```

Comment:
Suggestion: use "encontro" or "grupo de encontro" instead of "reunião" as a translation for meetup.

Reply:
Hi @nandaleite! So yes, "meetup" has the sense of an informal gathering (as in meetup.com). That's why I suggested the term "encontro", rather than "reunião" (compare "meeting").

Reply:
Thank you, @stormqueen1990! I added your suggestion in [5940466](https://github.com/kubernetes/website/pull/38407/commits/594046660a21c74d3eee374d60d87742b3c0a109)